driver/csiplugin/controllerserver.go

```diff
@@ -693,15 +692,22 @@ func (cs *ScaleControllerServer) GetVolIdMembers(vId string) (scaleVolId, error)
 	if len(splitVid) == 3 {
 		/* This is LW volume */
 		/* <cluster_id>;<filesystem_uuid>;path=<symlink_path> */
+		/* OR a static volume created for fileset */
+		/* <cluster_id>;<filesystem_uuid>;fileset=<filesetname> */
 		vIdMem.ClusterId = splitVid[0]
 		vIdMem.FsUUID = splitVid[1]
 		SlnkPart := splitVid[2]
 		slnkSplit := strings.Split(SlnkPart, "=")
 		if len(slnkSplit) < 2 {
 			return scaleVolId{}, status.Error(codes.Internal, fmt.Sprintf("Invalid Volume Id : [%v]", vId))
 		}
-		vIdMem.SymLnkPath = slnkSplit[1]
-		vIdMem.IsFilesetBased = false
+		if strings.EqualFold(slnkSplit[0], "fileset") {
```

Comment:
Hi Smita, Sorry I don't understand this part of change ? Can you please explain

Reply:
> Hi Smita, Sorry I don't understand this part of change ? Can you please explain why it is required

I think the (case insensitive) check is there to identify whether after filesystem UUID we have next argument as "fileset=..." or "path=..." and based on that set further variables.

Reply:
but this if is for LW volumes if len(splitVid) == 3 so there is never going to be case of fileset based volume. fileset volume handle splitting is taken care in next if where handle is of divided into 4 part.

Reply:
Right Deepak, this change was introduced as for fileset static provisioning the format was going to be **<cluster_id>;<filesystem_uuid>;fileset=<fileset_id>** without the "path". But it was later changed to also include path, hence reverting this change.